### ISSUE TITLE
api.getAccountData produces warning and return 0 balance.

### ISSUE THREAD

**Issue by srgg** [iotaledger/iota.lib.java#34]
I'm using testnet for PoC. I tried to get account data for my seed by using api.getAccountData, unfortunately it produces warning and return no balance and no transfers. In the log I found the following warning: 

```
19:30:00.810 [Parallel.For-0] WARN jota.IotaAPI - GetBundleError: 
jota.error.InvalidSignatureException: [Invalid Signatures!]
    at jota.IotaAPI.getBundle(IotaAPI.java:703)
    at jota.IotaAPI$1.perform(IotaAPI.java:198)
    at jota.IotaAPI$1.perform(IotaAPI.java:194)
    at jota.utils.Parallel$1.call(Parallel.java:44)
    at jota.utils.Parallel$1.call(Parallel.java:41)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)
```

It was called as follows:
```
 final GetAccountDataResponse accountData = api1.getAccountData(seed1, 1,0,true,0,true,0,0,true,0);
```

The same JS call works like a charm:
```
iota1.api.getAccountData(seed1, function(e, accountData){
    if (e) {
        console.log('Can\'t get balance for seed1 ', e);
    } else {
        console.log('Balanse for seed1 is: \n', accountData);
    }
});
```

and returns more adequate results:

```
{ latestAddress: 'NCBCUUKBMVOEKPEMLYWZDNGQVUIA9CLNJXPASMZXKSJIYYNLAVPZJIWMITBOHDXQSEHKZUITNVOMDQAJW',
  addresses: 
   [ 'XDLWWFWQUE9GBAAK9VCIYDLXPEHUHKQAPMSGML9GCDMTXFGBPXFYKCGTVVBF9JB9EQVBOXJFVEDQKUFGZ',
     'SGQBRPBYBUNCFD9ZCFMPGIRHLMXNZTKDPXYVDHKPRMZ9JE9G9XKSFHPEK9UENJRAZKOZPUIQ9CDA9AHYW',
     'SNNZ9RIZMQONXJARXUABJPZCFSJXXEYVPI9QPI9EXQVYYQTNALMUHUDIESBG9SLLWBXGL9LIAYEFAJIDA' ],
  transfers: 
   [ [ [Object], [Object], [Object], [Object] ],
     [ [Object] ],
     [ [Object], [Object], [Object], [Object] ] ],
  inputs: 
   [ { address: 'XDLWWFWQUE9GBAAK9VCIYDLXPEHUHKQAPMSGML9GCDMTXFGBPXFYKCGTVVBF9JB9EQVBOXJFVEDQKUFGZ',
       keyIndex: 0,
       security: 2,
       balance: 1000000000 } ],
  balance: 1000000000 }
```

I also checked api.getTransfers for both languages and found that the results are the same.

I'm not sure whether this warning valid or not, and why it works on JS and fails on Java,  but, definitely. if it affects the returning results it should be propagated rather than being swallowed.

**Comment by pinpong** [comment]
Is that issue fixed with v0.9.5?

**Comment by pinpong** [comment]
With #57 this issue should be closed.
